### PR TITLE
New adapter for APCU 5.1 (PHP 7)

### PR DESCRIPTION
**Pull request by remicollet**
See #47

### CODE REVIEW

File: src/Storage/Adapter/ApcuOptions.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Cache\Storage\Adapter;
+
+/**
+ * These are options specific to the APC adapter
+ */
+class ApcuOptions extends AdapterOptions
```
Comment (Maks3w) [inline src/Storage/Adapter/ApcuOptions.php]: This class could be removed and reuse ApcOptions

**Comment by Maks3w** [comment]
Please extend from APC classes and override the methods where needed.

**Comment by remicollet** [comment]
Please consider this mostly as a PoC.

I'm not enough aware of ZF internals to create something perfect, and don't want to spend my time bikesheeding.

At least, now, I know that it "can" work (with latest APCU 5.1 without apc compat ext), while it is for now broken.

Feel free to rewrite this as you prefer.

**Comment by Maks3w** [comment]
Thanks for the PoC. Unfortunately this can't be merged as is.

**Pull request closed without merging**